app.js

```diff
@@ -31,23 +31,55 @@ skylineDropdown.addEventListener('change', () => {
     skylineCount++; 
 
     skylineImageEl.src = `./assets/${skyline}-skyline.jpg`;
+    displayStats(); 
 });
 
 castleDropdown.addEventListener('change', () => {
     const castle = castleDropdown.value;
     castleCount++; 
 
     castleImageEl.src = `./assets/${castle}-castle.jpg`;
+    displayStats();
 });
 
 waterfrontDropdown.addEventListener('change', () => {
     const waterfront = waterfrontDropdown.value;
     waterfrontCount++;
 
     waterfrontImageEl.src = `./assets/${waterfront}-waterfront.jpg`;
+    displayStats();
 });
   // get user input
+sloganButton.addEventListener('click', () => {
+    const newSlogan = sloganInputEl.value;
+    sloganInput.push(newSlogan);
+
+    sloganInputEl.value = '';
```

Comment:
you change `sloganInputEl.vaue = ''` here in your event listener, and also in your display slogans function that you invoke on line 59 of your event listener.  Is this necessary both places?  Try it out and see!